vxWorks complains about building SPiiPlusDriver

> ../SPiiPlusDriver.cpp: In member function 'int SPiiPlusController::getNumAccelSegments(double)':
../SPiiPlusDriver.cpp:1480: error: 'lround' was not declared in this scope

and 

>../SPiiPlusDriver.cpp: In member function 'asynStatus SPiiPlusController::runProfile()':
../SPiiPlusDriver.cpp:1770: error: 'lround' was not declared in this scope
../SPiiPlusDriver.cpp:1819: error: 'lround' was not declared in this scope
../SPiiPlusDriver.cpp:1861: error: 'lround' was not declared in this scope

--- a/acsMotionApp/src/Makefile
+++ b/acsMotionApp/src/Makefile
@@ -8,7 +8,7 @@ include $(TOP)/configure/CONFIG
 
 DBD += AcsMotionSupport.dbd
 
-LIBRARY_IOC_Default = AcsMotion
+LIBRARY_IOC_DEFAULT = AcsMotion
 LIBRARY_IOC_vxWorks = -nil-
 
 SRCS += SPiiPlusBinComm.cpp

--- a/acsMotionApp/src/Makefile
+++ b/acsMotionApp/src/Makefile
@@ -8,7 +8,8 @@ include $(TOP)/configure/CONFIG
 
 DBD += AcsMotionSupport.dbd
 
-LIBRARY_IOC = AcsMotion
+LIBRARY_IOC_Default = AcsMotion
+LIBRARY_IOC_vxWorks = -nil-
 
 SRCS += SPiiPlusBinComm.cpp
 SRCS += SPiiPlusCommDriver.cpp

